src/compile/mark/text.ts

```diff
@@ -30,42 +35,69 @@ export namespace text {
       ['angle', 'align', 'baseline', 'dx', 'dy', 'font', 'fontWeight',
         'fontStyle', 'radius', 'theta', 'text']);
 
+    const config = model.config();
     const fieldDef = model.fieldDef(TEXT);
 
+    p.x = x(model.encoding().x, model.scaleName(X), config, fieldDef);
+
+    p.y = y(model.encoding().y, model.scaleName(Y), config, fieldDef);
+
+    p.fontSize = size(model.encoding().size, model.scaleName(SIZE), model.scale(SIZE), config);
+
+    p.text = text(model.encoding().text, model.scaleName(TEXT), config);
+
+    // FIXME: need this?
+    applyColorAndOpacity(p, model);
```

Comment:
Hmm.  I think this is a combo that does both color and opacity.  
I guess we can keep it for consistency with other marks?